model/views.py

```diff
@@ -0,0 +1,87 @@
+from django.shortcuts import render
+from django.views import generic
+
+from data.models import Contributor
+from openhub.models import (
+    PortfolioProject,
+    OutsideProject,
+    OutsideCommitter,
+    AffiliatedCommitter,
+    Organization
+    )
+
+
+def index(request):
+    args = {'portfolioprojects': 'portfolio_projects',
```

Comment:
always put dict members starting on the next line, and ALWAYS end EVERY row with a trailing comma, and the trailing `}` on a separate line.